TCAT/Controllers/HomeViewController.swift

```diff
@@ -509,7 +515,7 @@ extension HomeViewController: CLLocationManagerDelegate {
 // MARK: DZN Empty Data Set Source
 extension HomeViewController: DZNEmptyDataSetSource, DZNEmptyDataSetDelegate {
     func verticalOffset(forEmptyDataSet scrollView: UIScrollView!) -> CGFloat {
-        return -80
+        return tableView.tableHeaderView == nil ? -80 : (-80 + whatsNewView.frame.height + 20)
```

Comment:
Can you break this up and give descriptive variable names for what each means? You can still do ternary, but would help explain what's going on / make it less complex